### ISSUE TITLE
Project abandoned? 

### ISSUE THREAD

**Issue by davux** [citiususc/calendula#177]
Is Calendula still being developed? People are reporting issues but there are no responses other than the occasional "me too" from other users. 

There are also pull requests. 

If Calendula is not being developed anymore, it would be useful to state it clearly, and maybe someone will be interested in adopting it! It's a really great app.

**Comment by Midi-In** [comment]
I agree, as for an application I depend on all the time, it would be really nice to see the bugs fixed and features improved, might be worth stating the current project status officially and in case of abandonment someone could take the lead of the project by forking it

**Comment by angelpinheiro** [comment]
I have been one of the main developers of this project at CITIUS, but I am no longer part of the development team. Calendula is the open face of a larger project, in which I know there are people working. Hope the app can continue to live one way or another!

**Comment by alberto-illobre** [comment]
Greetings,

first of all, we would like to apologize for the delay in answering. We have been very busy developing a new version of Calendula for our Reginal Public Health Authority (SERGAS).

We have updated the project READMEs to reflect this. Please take a look. The good news is that we are expecting to push a bunch of bug fixes soon, and then update the Google Play version.

Thanks for your patience and regards,

The Calendula Team.

**Comment by hackel** [comment]
@aldoilsant Great news! I hope your team has had a chance to look at #136.  It is a critical bug that should definitely be fixed prior to any public deployment.

**Comment by ClearlyClaire** [comment]
Any update on the project?

**Comment by Nudin** [comment]
Seems like the app now isn't listed on google playstore anymore – does that mean it's finally time to consider it retired?
